demo/admin/src/products/categories/ProductCategoriesTable.tsx

```diff
@@ -98,14 +102,23 @@ const columns: GridColDef<GQLProductsCategoriesListFragment>[] = [
 ];
 
 function ProductCategoriesTable() {
+    const enableDragAndDrop = true;
+    const client = useApolloClient();
     const dataGridProps = { ...useDataGridRemote(), ...usePersistentColumnState("ProductCategoriesGrid") };
```

Comment:
I think you must disable paging if you use dnd

Reply:
I found a way to support it together with dnd. It's not perfect as you can not directly drag it to another page, but when switching between page sizes it's at least possible.

Reply:
ok, that works.

But I'm not sure if we should not radically remove features for grids with dnd sorting:
- remove paging
- remove sorting
- remove filtering (quick sort + field filters)

because
- you usually won't use dnd sorting for a large number of rows (>100)
- it's confusing to the user if it gets disabled when using filters
- the code gets simpler

Reply:
I think support for dnd with paging would be "nice to have". But I think it's ok to disable it in the first run and wait if anyone complains.

Should we disable paging by simply setting a high page-size (or imply a api-limit default) to fetch all possible, a fixed max like 100 or configurable via prop?

Reply:
it looks so empty without filters 🥲 

<img width="1078" alt="image" src="https://github.com/vivid-planet/comet/assets/1324250/3a7fbb50-5963-4e8a-8a78-1a359cb80d8d">

Reply:
> Should we disable paging by simply setting a high page-size (or imply a api-limit default) to fetch all possible, a fixed max like 100 or configurable via prop?

good question. I even thought about if api-generator should support non-paginated apis as well.

Reply:
> I even thought about if api-generator should support non-paginated apis as well.

if we got use cases like this, where paging is not supported, it would make sense...

Reply:
should adding PositionDecorator automatically disable paging (#2240)

Reply:
I'd say no. This seems like too much "magic" to me

Reply:
so this will be an admin-check only? because there is no technical reason api-vise to only allow position-feature if paging is disabled.